### ISSUE TITLE
Add a program using recursion

### ISSUE THREAD

**Issue by utkarsh-shekhar** [utkarsh-shekhar/basic-programs#11]
This is for git beginners.

Please add a program using recursion in the language of your choice. Make sure you add it under "recursion" folder for the language of your choice. Create that folder if it does not exist.
See readme for the structure.

Do let me know if anything is not clear.

**Comment by asdfMaciej** [comment]
I'll do it for Python

**Comment by utkarsh-shekhar** [comment]
@asdfMaciej Sure :)

**Comment by smoholkar** [comment]
is this still open ?

**Comment by utkarsh-shekhar** [comment]
@smoholkar Yes. Please check if there is no current pull request for the language of your choice. If there is no such pull request, feel free to raise one! :)

**Comment by Gayathrir1666** [comment]
Can we make pull requests in multi languages like one in c and the other in java and so.

**Comment by utkarsh-shekhar** [comment]
@Gayathrir1666 Yes. Just make sure that whatever you are working on has not already been done. Also check existing PRs for that.

**Comment by reedr7** [comment]
@utkarsh-shekhar  I see this issue is still open so,can I add more recursive programs apart from the factorial

**Comment by utkarsh-shekhar** [comment]
@reedr7 Yes, you can add any program which hasn't already been submitted (also check existing PRs).

**Comment by singloon** [comment]
@utkarsh-shekhar I have submitted a PR for kotlin recursion implementation of gcd, please review.
